source/api_documentation/call_ingestion_api/index.rst

```diff
@@ -0,0 +1,321 @@
+
+Calls submitted through this API are available to all post-call processing systems. This includes the Invoca transcription engine, Signal AI, as well as the Dashboard and Reporting system. 
+
+Invoca post-call processing services are applied to calls based on network and campaign settings.  The campaign for the call is determined by the *advertiser_campaign_id_from_network* parameter in the API.
+
+Additionally, Signals and Custom Data related to the call can be sumbitted during the initial call ingestion step.  Please note that Signals and Custom Data can also be applied in the future using the :doc:`../signal_api/index`.
+
+Signals can be any boolean value (e.g. sale, quote, etc), and Custom Data can be any alpha-numeric value (e.g. account type, customer quality score, etc).
+
+Response Codes
+--------------
+
+Remember to check the HTTP status code returned. This helps greatly when debugging.
+
+.. list-table::
+  :widths: 8 40
+  :header-rows: 1
+  :class: parameters
+
+
+  * - Status Code
+    - Meaning
+
+  * - 201 Created
+    - A new call creation request was successfully created.
+
+  * - 202 Accepted
+    - The call creation request has already been received and is currently being processed.
+
+  * - 400 Bad Request
+    - Attempted to make a request with an invalid API Version for route. Check the error message for any neccessary corrections
+
+  * - 401 Not Authorized
+    - Invalid or missing oauth token.
+
+  * - 403 Forbidden
+    - Attempted to access an invalid resource or provided invalid data. Check the errors object in the response.
+
+  * - 409 Conflict
+    - The **external_call_unique_id** in the request has already been used.  Please contact the Invoca support team at questions@invoca.com for further assistance.
+
+Request Parameters
+------------------
+
+Passed in “application/json” format.
+
+**Call Parameters**
+
+These are the call datails used when creating the call in the Invoca platform.
+
+    **Required**
+
+    `external_call_unique_id` The unique ID of the call in the system fom the external system.  This field is required to be unique across all calls within a network that are submitted from external sources.
+
+    `start_time` This is the date and time when the call started on the call origination platform.  Please see the **Timestamp Formats** section below for descriptions of supported timestamps.
+
+    `destination_phone_number` DNIS in E.164 format +country national_number; example: ‘+18885551212’. UK and Spanish numbers are also supported. Their country codes are +44 and +34 respectively.
+
+    `calling_phone_number` ANI in E.164 format +country national_number; example: ‘+18885551212’.
+
+    `advertiser_campaign_id_from_network` ID from network field on advertiser campaign; only calls within this advertiser campaign will be considered (campaign must be in the organization accessed via API).
```

Comment:
Instead of "only calls within this advertiser campaign will be considered", should we have something like "the submitted call will belong to this campaign"?

Reply:
Should we note here that the campaign must be configured as "External Only"? We have error documentation below stating that, but I think it would be helpful to know in this parameter list as well

Reply:
Good catch.  Thanks @kamerynn